Le monde 'maxMod5' diffère:
maxMod5(2,3) returned Ce test a levé une exception scala.MatchError : (2,3) (of class scala.Tuple2$mcII$sp) while 3 were expected.
maxMod5(6,2) returned Ce test a levé une exception scala.MatchError : (1,2) (of class scala.Tuple2$mcII$sp) while 6 were expected.
maxMod5(3,2) returned Ce test a levé une exception scala.MatchError : (3,2) (of class scala.Tuple2$mcII$sp) while 3 were expected.

